sass/MagicBox/MagicBox.scss

```diff
@@ -17,7 +17,7 @@
     > input {
       color: $grey-900;
       text-align: left;
-      padding: $padding 0;
+      padding: $padding $height+1 $padding 0;
```

Comment:
Should the right-padding be the width of the clear button rather than the $height? If we ever change the height of the searchbox, the alignment could break.

Reply:
It's a square, width = height, there's no width variable.